deploy/deploy-market.ts

```diff
@@ -20,6 +20,7 @@ export const deploy: DeployFunction = async (environment) => {
     hre,
     feeWallet,
     feePercentage: 15,
+    priceMultiple: 1,
```

Comment:
I think we need to have control down to at least a two decimal places.

Reply:
meaning we want to be able to set a price like $20.34 / NRT? 

to achieve this I guess we just need our price multiple to be scaled by 100 such that 1 is represented by 100 and 20.34 would be represented as 2034 and we divide by 100 after doing the calculations, and just make it very clear how this works in the documentation.  sound right?